Makefile

```diff
@@ -20,7 +20,7 @@ test: generate manifests
 	go test $(shell go list ./... | grep -v chaos-controller/controllers) -coverprofile cover.out
 
 # Run e2e tests (against a real cluster)
-e2e-test: generate manifests
+e2e-test: generate install
```

Comment:
This is the big change to debate. `install` is now required for e2e-test to work. We sort of always ran it first anyway though, but separately?

Reply:
I think it is an acceptable change.